src/PolyhedralGeometry/helpers.jl

```diff
@@ -133,6 +135,8 @@ homogenized_matrix(x::Union{AbstractVecOrMat,MatElem,Nothing}, val::Number) = ho
 dehomogenize(vec::AbstractVector) = vec[2:end]
 dehomogenize(mat::AbstractMatrix) = mat[:, 2:end]
 
+unhomogenized_matrix(x::Union{AbstractVecOrMat,MatElem}) = assure_matrix_polymake(x)
+
```

Comment:
Do we need this? Why not use `assure_matrix_polymake` directly?

Reply:
because we have different methods for `unhomogenized_matrix` for different input types, which in most cases do more than the method you were commenting on. we just let julia dispatch do the work for us. and we want the result to be `assured`.